packages/server-wallet/src/__test-with-peers__/crash-tolerance/crash-tolerance.test.ts

```diff
@@ -117,6 +97,9 @@ it('Create a directly-funded channel between two wallets, of which one crashes m
     turnNum: 0,
   });
 
+  await messageService.send(resultA2.outbox.map(o => o.params));
```

Comment:
TODO: Lots of `resultA2.outbox.map(o => o.params))`. Could be a helper or `send` could accept `o`